lib/stdlib/src/binary.erl

```diff
@@ -934,6 +935,30 @@ unhex(X) ->
              no, 10, 11, 12, 13, 14, 15, no, no, no, no, no, no, no, no, no  %96
             }).
 
+-doc """
+Joins a list of binaries together by a specified Separator.
+Equivalent to `iolist_to_binary(lists:join(Separator, Binaries))` with binary:join/2 being the faster of the two.
```

Comment:
```suggestion
Joins a list of binaries together by a specified `Separator`.

Equivalent to `iolist_to_binary(lists:join(Separator, Binaries))` with `binary:join/2` being the faster of the two.
```

Reply:
Just to clarify that the extra newline was intentional. The first paragraph is used as a summary for the function and (atleast IMO) the first sentence in this case is enough of a summary.

Reply:
Fixed once and for all now 😄